examples/basic-sc/README.md

```diff
@@ -50,107 +50,67 @@ objects would be set by the different personas in the system:
 
-```bash
-KAPP_CONTROLLER_VERSION=0.30.0
+- [knative-serving](https://knative.dev/docs/install/serving/install-serving-with-yaml/),
+  for being the runtime of the application we want to deploy.
 
-kapp deploy --yes -a kapp-controller \
-	-f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml
-```
+4. A container image registry in which you have authorization to push images. The
+  credentials will be passed to kpack in the configuration steps below.
 
-- [knative-serving], for being the runtime of the application we want to deploy.
+5. [Tree](https://github.com/ahmetb/kubectl-tree), a tool that we will use to observe the objects created.
 
-```bash
-KNATIVE_SERVING_VERSION=0.26.0
+## Running the example in this directory
 
-kapp deploy --yes -a knative-serving \
-  -f https://github.com/knative/serving/releases/download/v$KNATIVE_SERVING_VERSION/serving-crds.yaml \
-  -f https://github.com/knative/serving/releases/download/v$KNATIVE_SERVING_VERSION/serving-core.yaml
-```
+This example uses two directories with sub-directories of Kubernetes resources:
+[../shared](../shared) and [this directory (.)](.).
 
-4. Authorization to push images to a container image registry
+The shared directory has a subdirectory for cluster-wide configuration: [../shared/cluster](../shared/cluster)
 
-As mentioned before, in this example we make use of `kpack` to build container
-images out of the source code and push to a container image registry,
-so we must provide a Kubernetes secret that contains the credentials for pushing to a
-registry (see [kpack secrets] to know more about how kpack makes use of it).
+There is a subdirectory of cartographer-specific files that an App Operator would submit
+in both this and in the shared directory:
+[./app-operator](./app-operator) and [../shared/app-operator](../shared/app-operator)
 
+Finally, in this and in the shared directory there is a subdirectory containing
+Kubernetes objects that a developer would submit: [./developer](./developer) and
+[../shared/developer](../shared/developer)
 
-## Running the example in this directory
-
-As mentioned before, there are three directories: [./00-cluster](./00-cluster),
-for cluster-wide configuration, [./app-operator](./app-operator), with
-cartographer-specific files that an App Operator would submit, and
-[./developer](./developer), containing Kubernetes objects that a developer
-would submit (yes, just a [Workload] and a [Pipeline]!)
+### Configuring the example
 
 Before we get started with the details pertaining to Cartographer, first we
 need to set up a few details related to where container images should be pushed
 to - because [kpack/Image] is going to be used for building container images
 based on the source code, we need to tell it where those images should go, and
-which credentials it should use.
+which credentials it should use (see [kpack secrets] to know more about how
```

Comment:
I struggled a couple of times when running the example to actually understand that there is a requirement to create a secret to provide the necessary push permission. So probably making it a imperative statement: `Create a secret with the authentication details of the container registry you plan to use for the example see kpack docs (link) as a reference)`